src/fhirtypes/common.ts

```diff
@@ -58,6 +68,91 @@ export function setPropertyOnDefinitionInstance(
   setPropertyOnInstance(instance, pathParts, assignedValue, fisher);
 }
 
+export function createUsefulSlices(
+  instanceDef: StructureDefinition | ElementDefinition | InstanceDefinition,
+  instanceOfStructureDefinition: StructureDefinition,
+  ruleMap: Map<string, { pathParts: PathPart[] }>,
```

Comment:
After talking with @mint-thompson, we're actually okay with this since we only use the `pathParts` property, so it's fine as long as the `pathParts` is there, but we don't care about anything else.